### PR TITLE
3.1 release notes

### PR DESCRIPTION
**Pull request by David-Engel**

### CODE REVIEW

File: release-notes/3.1/3.1.0.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+This update includes the following changes over the 3.0 release:
+
+### Added
+
+- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1539](https://github.com/dotnet/SqlClient/pull/1539) [Read more](#introduce-attestation-protocol-none)
+- Included `42108` and `42109` error codes to retriable transient errors list. [#1560](https://github.com/dotnet/SqlClient/pull/1560)
+
+### Fixed
+
+- Changed EnclaveDelegate.Crypto GetEnclaveProvider to use a thread safe concurrent dictionary. [#1451](https://github.com/dotnet/SqlClient/pull/1451)
```
Comment (DavoudEshtehari) [inline release-notes/3.1/3.1.0.md:14]:
```suggestion
- Changed EnclaveDelegate.Crypto GetEnclaveProvider to use a thread safe concurrent dictionary. [#1564](https://github.com/dotnet/SqlClient/pull/1564)
```

**Review by DavoudEshtehari — COMMENTED** [review]
Could you update [README.md](https://github.com/dotnet/SqlClient/blob/main/release-notes/README.md) and [CHANGELOG.md](https://github.com/dotnet/SqlClient/blob/main/CHANGELOG.md), please?

File: CHANGELOG.md [hunk 2]
```diff
@@ -145,6 +145,17 @@ This update brings the below changes over the previous release:
...
+## [Stable release 3.1.0] - 2022-03-30
+
+### Added
+
+- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1419](https://github.com/dotnet/SqlClient/pull/1564) [#1425](https://github.com/dotnet/SqlClient/pull/1564)
+- Included `42108` and `42109` error codes to retriable transient errors list. [#1560](https://github.com/dotnet/SqlClient/pull/1560)
+
+### Fixed
+
+- Changed EnclaveDelegate.Crypto GetEnclaveProvider to use a thread safe concurrent dictionary. [#1451](https://github.com/dotnet/SqlClient/pull/1451)
```
Comment (DavoudEshtehari) [inline CHANGELOG.md:157]:
```suggestion
- Changed EnclaveDelegate.Crypto GetEnclaveProvider to use a thread safe concurrent dictionary. [#1564](https://github.com/dotnet/SqlClient/pull/1564
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -145,6 +145,17 @@ This update brings the below changes over the previous release:
...
 - Various code improvements [#902](https://github.com/dotnet/SqlClient/pull/902) [#925](https://github.com/dotnet/SqlClient/pull/925) [#933](https://github.com/dotnet/SqlClient/pull/933) [#934](https://github.com/dotnet/SqlClient/pull/934) [#1024](https://github.com/dotnet/SqlClient/pull/1024) [#1057](https://github.com/dotnet/SqlClient/pull/1057) [#1122](https://github.com/dotnet/SqlClient/pull/1122) [#1133]((https://github.com/dotnet/SqlClient/pull/1133)) [#1134](https://github.com/dotnet/SqlClient/pull/1134) [#1141](https://github.com/dotnet/SqlClient/pull/1141) [#1187](https://github.com/dotnet/SqlClient/pull/1187) [#1188](https://github.com/dotnet/SqlClient/pull/1188) [#1223](https://github.com/dotnet/SqlClient/pull/1223) [#1225](https://github.com/dotnet/SqlClient/pull/1225)  [#1226](https://github.com/dotnet/SqlClient/pull/1226)
 
+## [Stable release 3.1.0] - 2022-03-30
+
+### Added
+
+- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1419](https://github.com/dotnet/SqlClient/pull/1564) [#1425](https://github.com/dotnet/SqlClient/pull/1564)
```
Comment (DavoudEshtehari) [inline CHANGELOG.md:152]:
```suggestion
- Added new Attestation Protocol `None` for `VBS` enclave types. This protocol will allow users to forgo enclave attestation for VBS enclaves. [#1539](https://github.com/dotnet/SqlClient/pull/1539)
```

**Review by DavoudEshtehari — APPROVED** [review]

**Review by Kaur-Parminder — APPROVED** [review]

**Review by johnnypham — APPROVED** [review]

**Pull request merged**